ci/gpu/build.sh

```diff
@@ -192,13 +192,16 @@ else
     CONDA_FILE=${CONDA_FILE//-/=} #convert to conda install
     gpuci_logger "Installing $CONDA_FILE"
     conda install -c ${CONDA_ARTIFACT_PATH} "$CONDA_FILE"
-
+    
     gpuci_logger "Install the main version of dask and distributed"
     set -x
-    pip install "git+https://github.com/dask/distributed.git@main" --upgrade --no-deps
+    pip install "git+https://github.com/jakirkham/distributed.git@drop_whowhat_client" --upgrade --no-deps
```

Comment:
This can be reverted once ready as it is merged

```suggestion
    pip install "git+https://github.com/dask/distributed.git@main" --upgrade --no-deps
```